README.md

```diff
@@ -24,3 +24,21 @@ Or, if they're in a single tar.gz file:
 ```
 universalizer run -c graph.tar.gz
 ```
+
+### ID and category mapping
+
+SSSOM-format maps are supported. Use a single map file:
+
+```
+univeralizer run -m poro-mp-exact-1.0.sssom.tsv path/to/directory
+```
+
+or a whole directory of them:
+
+```
+univeralizer run -m path/to/mapfiles path/to/directory
+```
+
+For SSSOM maps from `subject_id` to `object_id`, subject node IDs will be remapped to object IDs.
+
+If the `object_id` is a Biolink category ID, however, the node's category ID will be remapped instead of its URI.
```

Comment:
Note that sssom has a `category` field which is suppose to be used for categories. I don't think it makes sense to have bl categories appearing in `object_id`

Reply:
I've been previously using `subject_id` and `object_id` to maintain the provenance of the category - if I just set `category` to what I want the category to be, that's fine, but it doesn't specify what category I expect it to be before the change.
That being said, it certainly seems to violate the intent of the standard! I'll use the `category` field for Biolink cat maps.

Reply:
I realized today that SSSOM expects both `subject_category` and `object_category` so that's not even a problem anymore.